src/Training.h

```diff
@@ -47,7 +47,7 @@ class TimeStep {
     using BoardPlane = std::bitset<NUM_INTERSECTIONS>;
     using NNPlanes = std::vector<BoardPlane>;
     NNPlanes planes;
-    std::vector<float> probabilities;
+    std::array<float, POTENTIAL_MOVES> probabilities{{0}};
```

Comment:
Why change the datatype here, especially if you're going to try to std::move it?

Reply:
NNPlanes and probabilities sizes are both fixed at compile time. Are there reasons to prefer std::vector over std::array here?